examples/statistics/boxplot_demo.py

```diff
@@ -149,39 +145,40 @@
         medianX.append(med.get_xdata()[j])
         medianY.append(med.get_ydata()[j])
         ax1.plot(medianX, medianY, 'k')
-        medians[i] = medianY[0]
```

Comment:
this line can be lifted out of the internal loop (and overwrites `medians`, as promised above).